src/components/MosaicAttachmentInput/MosaicAttachmentInputTs.ts

```diff
@@ -160,4 +165,7 @@ export class MosaicAttachmentInputTs extends Vue {
     public onMosaicAttachmentChange(mosaicAttachment: { mosaicHex: string; amount: string }) {
         this.relativeAmount = mosaicAttachment.amount;
     }
+    onCheckBalance(flag: boolean): void {
```

Comment:
Should we add some space(new line) between methods?

Reply:
done